src/i18n

```diff
@@ -1,3 +1,10 @@
+input database
```

Comment:
Probably, I updated it to use DDR tool for missing translation,, but you made a better tool since that I discover only after.